bundles/AdminBundle/config/aliases.yaml

```diff
@@ -2,7 +2,5 @@ services:
     _defaults:
         public: true
 
-    pimcore_admin.security.user_provider: '@Pimcore\Bundle\AdminBundle\Security\User\UserProvider'
     pimcore_admin.security.token_storage_user_resolver: '@Pimcore\Bundle\AdminBundle\Security\User\TokenStorageUserResolver'
-    pimcore_admin.security.user_loader: '@Pimcore\Bundle\AdminBundle\Security\User\UserLoader'
-    pimcore_admin.security.user_checker: '@Pimcore\Bundle\AdminBundle\Security\User\UserChecker'
\ No newline at end of file
+    pimcore_admin.security.user_loader: '@Pimcore\Bundle\AdminBundle\Security\User\UserLoader'
```

Comment:
why keep these 2 aliases?

Reply:
done in https://github.com/pimcore/pimcore/pull/14797/commits/6a8d1d16d8ef0e0c33f8474c8a03c110a64dad8c. 

The `pimcore_admin.security.token_storage_user_resolver` alias is needed here

https://github.com/pimcore/customer-data-framework/blob/e2e63de9469fabaa3445bc6a14ca502bc953a9d8/src/Newsletter/ProviderHandler/Mailchimp/UpdateFromMailchimpProcessor.php#L31

Reply:
@Corepex we can inject it in the `UpdateFromMailChimpProcessor` on latest version of CMF, as it is only supported on Pimcore 11.

Reply:
Oke lets do that :) I will create a followup PR for CMF

Reply:
created a PR to inject the service https://github.com/pimcore/customer-data-framework/pull/441